packages/suite/src/components/suite/layouts/LoggedOutLayout.tsx

```diff
@@ -30,28 +31,30 @@ export const LoggedOutLayout = ({ children }: LoggedOutLayout) => {
     useClearAnchorHighlightOnClick(wrapperRef);
 
     return (
-        <Wrapper ref={wrapperRef}>
-            <PageWrapper>
-                <ModalContextProvider>
-                    <Metadata title={title} />
-                    <ModalSwitcher />
-
-                    <LayoutContext.Provider value={setLayoutPayload}>
-                        <Body data-test="@suite-layout/body">
-                            <Columns>
-                                <AppWrapper data-test="@app" ref={scrollRef} id="layout-scroll">
-                                    {TopMenu && <TopMenu />}
-
-                                    <ContentWrapper>{children}</ContentWrapper>
-                                </AppWrapper>
-                            </Columns>
-                        </Body>
-                    </LayoutContext.Provider>
-
-                    {!isMobileLayout && <GuideButton />}
-                </ModalContextProvider>
-            </PageWrapper>
-            <GuideRouter />
-        </Wrapper>
+        <ElevationContext baseElevation={-2}>
```

Comment:
Top level of the app is `-2` so it next `useElevation` returns `-1` which is lowest visible level